client/src/pages/profile.js

```diff
@@ -342,4 +326,28 @@ export default withStyles({
       </div>
     )
   }
-})
+}
+
+export default withStyles({
+  quote: {
+    fontSize: 'small',
+    overflowWrap: 'break-word'
+  },
+  icon: {
+    '& svg': {
+      verticalAlign: 'middle',
+      height: '20px',
+      fill: '#333'
+    },
+    marginRight: '25px'
+  },
+  form: {
+    '& button': {
+      margin: 0,
+      lineHeight: '20px',
+      padding: '10px',
+      float: 'right'
+    },
+    padding: '0 !important'
```

Comment:
Could we add an option to the `Form` component instead of overriding the padding using `!important`?

Reply:
Created an issue, seems out of scope of this PR